### ISSUE TITLE
Open Source Backend

### ISSUE THREAD

**Issue by richcouzzi** [krasimir/demoit#13]
@krasimir thanks for making this—have been having fun using it but realize the limitations while in running dev mode (or running at localhost:8004). **Would you be open to open sourcing the backend** or providing a swagger doc or similar so those that are interested can roll their own? Namely, the ability to export to zip seems a prod-only feature.

Somewhat related: **I am unable to login via Github on https://poet.krasimir.vercel.app** — when looking at Github connected oauth applications, I do see Poet.codes, thought it lists `https://poet.krasimir.now.sh`—I see the other URLs were migrated from `now.sh`, but not this one. Perhaps this is related? FWIW, error message reads: "Authentication failed   Reason: ML"

I understand this is an old repo and you might not be interesting in working on it. If that's the case, no worries at all. 
Just to say—**thanks again, for this and your consideration.**

**Comment by krasimir** [comment]
Hey @richcouzzi,

the only backend that I wrote for this tool exists in the Poet project. I don't mind sharing it. Here it is the whole site, editor and its backend -> https://krasimirtsonev.com/code/poet

Poet was kinda big thing for me when I worked on it. There was this integration with GitHub, the ability to save "stories" in a database (I used MongoDB atlas) and indeed it was hosted on Zeit (now Vercel). Sadly the project didn't take off so I slowly abandon it. Feel free to read and use the code above. I don't know much about it so I can't help with direction on what is what :) I worked on that project ~4 years ago so I have pretty vague memory of how the things work.

P.S.
I just tried to the current version at poet.krasimir.vercel.app but sadly my hobby plan on Vercel is not enough for the amount of endpoints that I have in my `now.json` file.
